src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java

```diff
@@ -237,6 +240,11 @@ private void initCatalogsTab() {
                 })
                 .install(catalogTable);
 
+        Set<String> selectedFetchers = prefs.getWorkspacePreferences().getSelectedSlrFetchers();
+        for (StudyCatalogItem item : catalogTable.getItems()) {
+            item.setEnabled(selectedFetchers.contains(item.getName()));
+        }
+
```

Comment:
Move this into the preferences. The View file should only contain bindings. For every logic part the viewmodel is responsible, because the viewmodel can and should be tested.
This means: Create a property in the viewModel: selectedFetchersProperty, create a getter for that property, create a binding in the view between the ui component and the getter of the viewmodel for that property.

Reply:
Sorry, not preferences. I meant ViewModel.